engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/UpdateByOperatorFactory.java

```diff
@@ -1304,9 +1310,9 @@ private UpdateByOperator makeRollingWAvgOperator(@NotNull final MatchPair pair,
 
             final String[] affectingColumns;
             if (rs.revWindowScale().timestampCol() == null) {
-                affectingColumns = new String[] {pair.rightColumn, rs.weightCol()};
+                affectingColumns = new String[] {pair.rightColumn};
```

Comment:
Weighted average needs to care about weights, right? Why isn't this failing a unit test?